[msbuild] triplet installation folder is wrong since tag 2021.12.01

**Describe the bug**
_scripts\buildsystems\msbuild\vcpkg.targets_ creates the the triplet installation folder variable `_ZVcpkgCurrentInstalledDir` in a wrong way - the resulting path contains two path separators, which Windows 11 doesn't recognize as a correct path.

**Environment**
- OS: Windows 11

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg install libxml2:x64-windows
2. ./vcpkg export libxml2:x64-windows --nuget --nuget-id=vcpkg-export --nuget-version=1.0.0
3. Consume .nuget package in Visual Studio solution/project

As per my [comment 1](https://github.com/microsoft/vcpkg/commit/97e7ac8d748779d04ab1c230abb59f4473a227ed#r62142223) and [comment 2](https://github.com/microsoft/vcpkg/pull/19767/files#diff-705430a4414b12ed477c77d25c696237d3936f152c0c63a4c8e240bbeb5dc3bbR65) the PR #19767 introduced a double-backslash, which leads to 'No such file or directory' compiler error messages.

**Expected behavior**
Because `_ZVcpkgInstalledDir` already has a trailing backslash, `_ZVcpkgCurrentInstalledDir` should be constructed like this: `$(_ZVcpkgInstalledDir)$(VcpkgTriplet)`, such that resulting the triplet installation folder path contains only single backslashes.

**Failure logs** 

> 1>Target VcpkgTripletSelection:
> 1>  Using triplet "x64-windows" from "<...>\packages\vcpkg-export.1.0.0\installed\\\\x64-windows\"
> 1>Target ClCompile:
> 1>  stdafx.cpp
> 1>  stdafx.h(57,10): fatal error C1083: Cannot open include file: 'libxml/tree.h': No such file or directory

--- a/scripts/buildsystems/msbuild/vcpkg.targets
+++ b/scripts/buildsystems/msbuild/vcpkg.targets
@@ -62,7 +62,7 @@
   </Choose>
 
   <PropertyGroup>
-    <_ZVcpkgCurrentInstalledDir>$(_ZVcpkgInstalledDir)\$(VcpkgTriplet)\</_ZVcpkgCurrentInstalledDir>
+    <_ZVcpkgCurrentInstalledDir>$(_ZVcpkgInstalledDir)$(VcpkgTriplet)\</_ZVcpkgCurrentInstalledDir>
     <_ZVcpkgNormalizedConfiguration Condition="$(VcpkgConfiguration.StartsWith('Debug'))">Debug</_ZVcpkgNormalizedConfiguration>
     <_ZVcpkgNormalizedConfiguration Condition="$(VcpkgConfiguration.StartsWith('Release')) or '$(VcpkgConfiguration)' == 'RelWithDebInfo' or '$(VcpkgConfiguration)' == 'MinSizeRel'">Release</_ZVcpkgNormalizedConfiguration>
 

